### DISCUSSION TITLE
Saved variants with custom tag

### DISCUSSION THREAD

**Discussion by bnouyou** [broadinstitute/seqr#2155] [Q&A]
Hello,

I work in a local instance of seqr. I have created custom tags for the variants. They have been used by the biologists to tag some variants.
I would like to konw if it's possible to have them in the "saved variants" ? As far as I can see, in "Summary Data" > "Saved Variants", in "Showing" (top-left) I can select tags. But I don't have my custom tags.
Moreover, In the "Saved variants" page, I don't see a lot of variants, so I am wondering if they have been saved. How this is working ? Each time someone tag a savriant, is it saved ?

Thank you in advance for your help.

**Comment by hanars** [comment]
The  "Summary Data" > "Saved Variants" page is designed for being able to pull data from all seqr projects for exporting/ reporting, and only supports a limited number of tags for this purpose. For day-to-day viewing and reviewing of variants, you should be looking at the "saved variants" page for a specific project or family. There, you will be able to see variants with any tag including custom tags
